doc/circuitikzmanual.tex

```diff
@@ -2485,6 +2485,8 @@ \subsection{Block diagram components}
     \circuitdesc*{wilkinson}{wilkinson divider}{}( in/180/0.1, out2/45/0.1, out1/-45/0.1 )
     \circuitdesc*{gridnode}{gridnode\footnotemark}{}(left/135/0.2, right/45/0.2, center/-100/0.4, up/90/0.2, down/-45/.2)
     \footnotetext{added by \texttt{olfline}}
+    \circuitdesc*{mzm}{Mach Zehnder Modulator\footnotemark}{} ( in/180/0.1, mod/90/0.1, out/0/0.1)
```

Comment:
You have to remove the space before the `(`, otherways it will not work:

![image](https://user-images.githubusercontent.com/6414907/88031513-fd843e00-cb3c-11ea-9872-9e1cd9393c76.png)